test/pages/IndDoYouHaveNINumberPageSpec.scala

```diff
@@ -33,6 +36,17 @@ class IndDoYouHaveNINumberPageSpec extends PageBehaviours {
     booleanField  <- arbitrary[Boolean]
   } yield (addressLookup, address, postcode, name, booleanField)
 
+  private val addressResponseGen: Gen[AddressResponse] = for {
```

Comment:
An AddressResponse generator already exists in test-utils/ModelGenerators - line 149 (it's imported as RegistrationAddressResponse which may have made it evade Cmd+F)

It uses the Address generator at its code so could you update that instead please then it should fix all address for all pages.